build/jest/enzymeAdapter.js

```diff
@@ -0,0 +1,25 @@
+import Enzyme, { mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+// make Enzyme functions available in all test files without importing
+global.shallow = shallow;
+global.mount = mount;
+
+if (window.document) {
```

Comment:
I'll move this out to another env-file, since it doesn't really have anything to do with Enzyme.